ci/windows/test.cmd

```diff
@@ -1,7 +1,11 @@
 :: See build.cmd for documentation on this call.
 call "c:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
 
-:: We currently don't have any tests to run on Windows, so this is just commented out.
-:: We'll expand on this later.
-:: cd build
-:: ctest -C release || exit \b 1
+cd build
+
+:: This sets up ZEEKPATH and ZEEK_PLUGIN_PATH
+call zeek-path-dev.bat
+
+:: Only run the unit tests for now. Btest is supported on Windows but a ton
+:: of tests are still failing so it's not worth trying to run it.
+src\zeek --test
```

Comment:
Does this have the same rules as in a bash/sh that the exit code of test.cmd is that of the last command executed within?